### PR TITLE
excluded old info and referenced new one

### PR DESCRIPTION
**Pull request by VLG17**
https://github.com/MicrosoftDocs/windows-itpro-docs/issues/4527

### CODE REVIEW

**Comment by mypil** [comment]
@nenonix @JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: windows/client-management/mdm/management-tool-for-windows-store-for-business.md [hunk 1]
```diff
@@ -81,43 +81,7 @@ For code samples, see [Microsoft Azure Active Directory Samples and Documentatio
...
-
-10. Create a client key.
-
-    ![business store management tool](images/businessstoreportalservices14.png)
-
-    > **Note**  In the prior version of the tool, an update to the app manifest was required to authorize the application. This is no longer necessary.
-     
-11. Login to Store for Business and enable your application. For step-by-step guide, see [Configure an MDM provider](https://technet.microsoft.com/library/mt606939.aspx).
-
+[Here](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app) are the steps to configure your Azure AD app.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/management-tool-for-windows-store-for-business.md]:
```suggestion
See [Quickstart: Register an application with the Microsoft identity platform](https://docs.microsoft.com/en-us/azure/active-directory/develop/quickstart-register-app) for the steps to configure your Azure AD app. 
```

Comment (mypil) [reply windows/client-management/mdm/management-tool-for-windows-store-for-business.md]: @VLG17 - please also remove the "en-us" from the link.

Comment (illfated) [reply windows/client-management/mdm/management-tool-for-windows-store-for-business.md]: @JohanFreelancer9 : Looks like you went a little too far there by cutting `/en-us/azure/` down to only `/zure/` -- you may want to edit that suggestion (unless my page is shown incorrectly at the moment).

Comment (JohanFreelancer9) [reply windows/client-management/mdm/management-tool-for-windows-store-for-business.md]: @illfated Thank you!

**Comment by mypil** [comment]
@ManikaDhiman - PR has been copy edited and is ready for final review. This is based on issue #4527.

Thank you.

**Pull request merged**